dirsrvtests/tests/suites/clu/dsidm_config_test.py

```diff
@@ -0,0 +1,216 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2020 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+import time
+import subprocess
+import pytest
+
+from lib389.cli_idm.client_config import *
+from lib389.tasks import *
+from lib389.utils import *
```

Comment:
Could you, please, specify the exact imports here? 

`import *` is too broad and it's a bad coding habit we better get rid of from our source :)